rma/models/rma_operation.py

```diff
@@ -10,7 +13,145 @@ class RmaOperation(models.Model):
 
     active = fields.Boolean(default=True)
     name = fields.Char(required=True, translate=True)
+    color = fields.Integer()
+    count_rma_draft = fields.Integer(compute="_compute_count_rma_draft")
```

Comment:
Better to do this in one compute, and with `read_group` by `state`, obtain all the numbers and sum them according your needs.

Reply:
done